ambient.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'command-exists'
```

Comment:
hello, FYI I removed in previous PR the listr entry and used @types/listr package instead (and all this file BTW)
is that this file entries are added automatically somehow ?
because with `@types/<module>` packages we don't need it that file

Reply:
I added it back because `@types/<module>` hasn't been [updated since a while](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27562/files) and I am using the `collapse` option that made tsc complaining. I don't think we need `command-exists` and `listr-renderer` though and I will remove them.